test/messages/commands/index.js

```diff
@@ -142,7 +142,9 @@ describe('Command Messages', function() {
 
     it('should error if filter is not a bloom filter', function() {
       (function() {
-        var message = messages.FilterLoad({filter: 'not a bloom filter'});
+        var message = messages.FilterLoad({
+          filter: 'not a bloom filter'
+        });
```

Comment:
None, it's part of JS formatter I use: https://www.npmjs.com/package/js-beautify